Kickstarter-iOS/Views/ProjectNavigationSelectorView.swift

```diff
@@ -268,10 +265,7 @@ final class ProjectNavigationSelectorView: UIView {
 private let bottomBorderViewStyle: ViewStyle = { view in
   view
     |> \.backgroundColor .~ .ksr_support_100
-    |> \.layer.shadowColor .~ UIColor.ksr_black.cgColor
-    |> \.layer.shadowOpacity .~ ProjectNavigationSelectorViewStyles.Layout.shadowOpacity
-    |> \.layer.shadowOffset .~ ProjectNavigationSelectorViewStyles.Layout.shadowOffset
-    |> \.layer.shadowRadius .~ ProjectNavigationSelectorViewStyles.Layout.shadowRadius
+    |> dropShadowStyle()
```

Comment:
Unrelated to this PR but discovered it while I was doing this and decided to add it.